diff --git a/bin/gitcracken-patcher.ts b/bin/gitcracken-patcher.ts
index bade27b..0a91082 100644
--- a/bin/gitcracken-patcher.ts
+++ b/bin/gitcracken-patcher.ts
@@ -26,12 +26,6 @@ async function executeActions(actions: Actions[]) {
   for (const action of actions) {
     switch (action) {
       case Actions.backup: {
-        console.log(
-          `${chalk.green("==>")} ${emoji.get("package")} Backup ` +
-            `${chalk.green(patcher.asar)} ➔ ${chalk.green(
-              patcher.backupAsar(),
-            )}`,
-        );
         break;
       }
       case Actions.unpack: {
